client/features/media/MediaDetail.jsx

```diff
@@ -25,7 +25,8 @@ class MediaDetail extends React.Component {
       imgUrl: '',
       userSubs: [],
       watchWithSubscribed: [],
-      watchWithUnsubscribed: []
+      watchWithUnsubscribed: [],
+      username: 'lil timmy'
```

Comment:
NIce, thanks! I see you state for all your other properties with getMediaAndUserDetails(). Are you going to set the username state with the same function? Or maybe it's easier to inherit from a parent. If inherit from parent you can set the username with componentDidUpdate(). Just a couple options to consider